manifests/iaas/openstack.yml

```diff
@@ -1,10 +1,10 @@
 ---
 params:
   openstack_auth_url: (( param "What is the OpenStack Auth URL?" ))
-  openstack_username: (( vault meta.vault "/openstack/creds:username" ))
```

Comment:
This error is my fault, because while your fix is correct for the kit.yml, the kit.yml is incorrect in regards to the authoritative hooks/new (because the provided secrets are extracted from that script):

https://github.com/genesis-community/bosh-genesis-kit/blob/master/hooks/new#L203
https://github.com/genesis-community/bosh-genesis-kit/blob/master/hooks/new#L211-L214

The fix should be in kit.yml to add the `/creds` suffix to the path component, so that existing deployments will continue to work.